low-level/sspl_ll_d

```diff
@@ -254,6 +255,25 @@ def get_sensor_thread_deps(name, sspl_threaded_modules):
             deps.append(n)
     return deps
 
+def get_recovery_config(module_name):
```

Comment:
one more blank line above.

Reply:
Removed this method from this file and moved to thread_controller.py

Reply:
Done.